addons/l10n_sa_edi/data/ubl_21_zatca.xml

```diff
@@ -214,6 +216,33 @@
                 </cac:InvoiceLine>
             </xpath>
 
+            <!-- For ZATCA, we do not use RequestedMonetaryTotal tags. We always use LegalMonetaryTotal. -->
+            <xpath expr="//*[local-name()='RequestedMonetaryTotal']" position="replace">
```

Comment:
I'm not sure but can't you just modify the `t-if` instead ? Because you can do an xpath on a `t-if` and edit its attributes :)
search the codebase with the regex to find examples: `t-if.*attrib`

Reply:
I'm not sure how? I need to remove this tag completely in this case.

Reply:
```xml
<xpath expr="//*[@name='legal_monetary_total']" position="attributes">
    <attribute name="t-if">False</attribute>
</xpath>
<xpath expr="//*[@name='requested_monetary_total']" position="attributes">
    <attribute name="t-if">False</attribute>
</xpath>
```
Should work after naming the `t-if` nodes in the `ubl_20_templates.xml` :)

Reply:
Not needed anymore with the inheritance